docs/source/release/index.rst

```diff
@@ -12,6 +12,7 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
 
+* :support:`2505` Remove deprecated `ibis.HDFS`, `ibis.WebHDFS` and `ibis.hdfs_connect`
```

Comment:
can you put a marker in the docs to indicate where this version starts (and also remove the text above this as i think obsolete)

Reply:
Aren't the version headers added automatically by the script? There are no headers in the file for previous versions.

And I think the header above is still relevant, release notes before 1.0 are still in that other file.

Reply:
we should just re-integrate the docs, having this is sort of odd (e.g. make an extended whatsnew like pandas is much better so that these are in one place), but ok.

Reply:
I'm happy to merge both files if that's what you propose. Not sure why it's splitted. Also ok to have a separate file for each version if you prefer that.

If you can clarify whether:
- You want to continue like now, but with the pre 1.0 file merged into this one
- Have a separate file for each version and remove the sphinx extension that automatically adds the headers

I'll create an issue for it.